packages/run-npm/src/index.ts

```diff
@@ -3,10 +3,10 @@ import path from 'path'
 import spawn from 'cross-spawn'
 import PATH from 'path-name'
 
-export default function runNpm (npmPath: string | undefined, args: string[]) {
+export default function runNpm (npmPath: string | undefined, args: string[], cwd?: string) {
```

Comment:
usually we add `opts` if there are more than 2 arguments in a function

Reply:
I did it this way to minimize the changes of the PR, wouldn't that mean adapt everywhere "runNpm" is used?

Reply:
opts can be optional too

Reply:
oh ok, the third argument as an options object, I thought you meant including "args", I will do it 👍